content/events/fi/instabots-and-cybersecurity/index.md

```diff
@@ -9,12 +9,12 @@ type: "event"
 lang: fi
 ---
 
-In January we visited Identio for a meetup about Instagram bots and Cybersecurity.
+Tammikuussa kokoonnuimme Idention tiloihin puhumaan Instagramista ja kyberturvallisuudesta.
 
-Heta Pärssinen is a fullstack developer working at Duunitori. Building bots for Instagram has been her occasional hobby and she wanted to share the technical secrets with us at this meetup.
+Heta Pärssinen työskentelee fullstack -kehittäjänä Duunitorilla. Hän on harrastuksekseen rakentanut Instagram botteja, ja halusi jakaa miitissämme niiden teknisiä salaisuuksia.
 
 ![Heta presenting about Instabots.](heta.jpg)
```

Comment:
A small note: These texts are alt-texts for the images, so could you also translate them? 😊

Reply:
Hi, I translated now the texts that were in square brackets but should I make a new pull request now or how are the new changes transferred to the file?

Reply:
They have been automatically pushed to this PR, so no need to make anything more! 😊